File: Practice/AllDomains/Tutorials/30DaysOfCode/Day2Operators.py
/*	author:@shivkrthakur  */
mealCost =float(input())
tipPercent = float(input())
taxPercent = float(input())

tip = mealCost * (tipPercent/100)
#print "Tip: ", tip
tax = mealCost * (taxPercent/100)
#print "Tax: ", tax
totalCost = mealCost + tip + tax
#print "TotalCost: ", totalCost
totalCost = int(round(totalCost))
print("The total meal cost is", totalCost,"dollars.")

File: Practice/AllDomains/Tutorials/30DaysOfCode/Day2Operators.js
/*	author:@shivkrthakur  */
'use strict';

var _input = '';
var _index = 0;
process.stdin.on('data', (data) => { _input += data; });
process.stdin.on('end', () => {
	_input = _input.split(new RegExp('[\n ]+'));
	main(+(_input[0]), +(_input[1]), +(_input[2]));
});

function main(mealCost, tipPercent, taxPercent) {
  //console.log([ mealCost, tipPercent, taxPercent ].join(' '));
  var tip = tipPercent/100 * mealCost;
  var tax = taxPercent/100 * mealCost;
  var totalCost = mealCost + tip + tax;
  totalCost = Math.round(totalCost);
  console.log("The total meal cost is %d dollars.", totalCost);
}
